CMakeLists.txt

```diff
@@ -15,12 +15,15 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 macro(enable_strict_compiler_warnings target)
-  if (MSVC)
-    target_compile_options(${target} PRIVATE /WX /W4)
-  elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    target_compile_options(${target} PRIVATE -Wall -Wextra -Wpedantic -Werror -Wold-style-cast -Wfloat-equal -Wmost -Wunused-exception-parameter)
-  else()
-    target_compile_options(${target} PRIVATE -Wall -Wextra -Wpedantic -Werror -Wold-style-cast -Wfloat-equal)
+  # Disable warnings if USE_FOXGLOVE_SDK is enabled until SDK with fixed warnings is released
```

Comment:
🤔   not sure what the warning issues you ran into were, but if the SDK is added to include directories with the SYSTEM option, that may silence warnings from the SDK's own headers (although we should still try and fix those!)